SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security
+
+*Miden is currently in public testnet. Private reports help us fix security issues before mainnet launch.*
```

Comment:
Is there value for security issues to be reported privately (vs. publicly like other issues)?